setup.py

```diff
@@ -46,6 +46,7 @@
         'numba >= 0.43.0',
         'soundfile >= 0.9.0',
     ],
+    python_requires='>=3.6, <=3.8',
```

Comment:
<=3.8 fails on, say, 3.8.2.  (Two of our CI builds failed from this.)

Can we drop the upper bound?

Otherwise, LGTM.

Reply:
I see. I thought it would take all 3.8 versions. That's my mistake.

This is really more of an opinion than anything. If you've tested on a specific set of python interpreters then you can think of python_requires as a statement about what versions are validated to work with librosa. OTOH, you can treat future breaking changes in the interpreter as exceptional and assume forwards compatibility. My preference is for the defensive approach, which is probably an artifact of living in dependency hell so often. But if you think most librosa users are in a more researchy environment then I can see the argument for assuming forwards compatibility.

Should I drop the upper bound in the PR? Or bump the 3.8 version higher (e.g 3.8.2)?

Reply:
I'd prefer to drop the upper bound. Python releases more frequently than we do, and there's nothing on the 3.9 roadmap that should break compatibility for us anyway.